src/rust/aggregator/src/data/user_history.rs

```diff
@@ -288,6 +293,8 @@ impl Data for UserHistory {
                             &change.order_id,
                             &change.market_id,
                             &change.time,
+                            &fill.txn_version,
+                            &fill.event_idx,
```

Comment:
@CRBl69 It doesn't look like you de-duplicate fill events in this code here. There two fill events emitted for every fill, one for the maker and one for the taker. You can de-duplicated them using the order id field (I believe).